server/src/test/java/org/opensearch/identity/UserIdentityTests.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.identity;
+
+import org.opensearch.identity.UserIdentity.AnonymousUser;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNot.not;
+
+public class UserIdentityTests {
+
+    public void testGetIds() {
+        final String idOfSally = "Sally";
```

Comment:
Alice might be annoyed that Bob is now hanging out with Sally.

More seriously, I'd like to get away from using names as "ID" strings.  Particularly if we use SCIM there's a separate name field/fields.

Reply:
I think that tests are easier to read with these simpler strings even if the standard representation is a uuid.  Note; another great reason to use a library not having to author these tests.